core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -6,6 +6,7 @@
 
 package org.opensearch.sql.expression.datetime;
 
+import static java.time.temporal.ChronoField.YEAR;
```

Comment:
is there no YEAR field in ExprCoreType?

Reply:
No.. Why it should be? Aren't you messing `ChronoField` and `ExprCoreType`? `ExprCoreType` is a enum of `STRING`, `DOUBLE`, etc